userspace/libsinsp/parsers.cpp

```diff
@@ -4768,7 +4768,8 @@ void sinsp_parser::parse_container_evt(sinsp_evt *evt)
 	parinfo = evt->get_param(3);
 	container_info.m_image = parinfo->m_val;
 
-	m_inspector->m_container_manager.add_container(container_info, evt->get_thread_info(true));
+	auto container = std::make_shared<sinsp_container_info>(container_info);
```

Comment:
Looks like there is one more copy that could be fixed here in `sinsp_parser::parse_container_evt`